src/main/java/com/powsybl/openloadflow/ac/AbstractTransformerVoltageControlOuterLoop.java

```diff
@@ -47,4 +51,41 @@ protected OuterLoopStatus roundVoltageRatios(OuterLoopContext context) {
         }
         return status;
     }
+
+    public static void checkControl(LfNetwork network) {
+        var connectivity = network.getConnectivity();
```

Comment:
I'm wondering if using the DecrementalConnectivity here is consistent, as the init is costly. 

If we launch a security analysis, that's not a question as the init will be reused.

But if we only launch a loadflow, that's not really appropriate, and the naive approach is 4 times faster.

Reply:
If we keep it this way, the performance issue has been fixed by #537. It was because we get the main connected component many times. And EvenShiloach algorithm recomputes the main connected component each time it is asked (it was first implemented for use cases where only the small components were needed).

Reply:
> I'm wondering if using the EvenShiloachGraphDecrementalConnectivity here is consistent, as the init is costly.
> 
> If we launch a security analysis, that's not a question as the init will be reused.
> 
> But if we only launch a loadflow, that's not really appropriate, and the naive approach is 4 times faster.

This piece of code is ran both for a simple LF and a SA. We probably should use naive implementation for the LF and EvenShiloach for the SA but in both cases it has to be fast so it's a good thing that you improved performance for EvenShiloach.